### PR TITLE
Docs correct installation process instructions

### PR DESCRIPTION
**Pull request by Szarad**
Modify the installation instruction and add information for Mac users about env variables running locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,40 +3,83 @@
...
+pipenv run python3.8 -m spacy download en_core_web_sm
+pipenv run python3.8 -m spacy download de_core_news_sm
+```
+
+## using virtual environment (venv)
+
+```
+python3.8 -m venv /path/to/new/virtual/environment
+python3.8 -m source /path/to/new/virtual/environment/bin/active
+python3.8 -m pip install -r requirements.txt
```
Comment (lsmith77) [inline README.md:33]: is there a way to convert a Pipefile(.lock) to a requirements.txt file?

Comment (Szarad) [reply README.md:33]: Yes, I used it. I add this step.

**Review by lsmith77 — APPROVED** [review]

**Comment by lsmith77** [comment]
LGTM .. merge merge :)

File: README.md [hunk 2]
```diff
@@ -3,40 +3,84 @@
...
 see https://docs.platform.sh/development/cli.html for details
 
 ## Run Locally
 
+---
+
+Note for the Mac users. Set environment variables with the following snippet:
+
+```
+source .env.development.mac
```
Comment (lsmith77) [inline README.md:74]: ah one thing .. did you see that pydantic supports dotenv?

https://pydantic-docs.helpmanual.io/usage/settings/#dotenv-env-support

Comment (Szarad) [reply README.md:74]: Yes, I saw it. It can be changed to load from file instead of env variables.

**Pull request merged**